requests_toolbelt/multipart/encoder.py

```diff
@@ -339,7 +339,7 @@ class MultipartEncoderMonitor(object):
                                        MultipartEncoderMonitor)
         import requests
 
-        def callback(encoder, bytes_read):
+        def callback(encoder):
```

Comment:
Should we rename this to `monitor` instead of `encoder` to make it yet clearer what instance you're actually getting?